code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/util/DistanceCalculationUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.research.code.submissions.clustering.util
```

Comment:
Please, move intro the org.jetbrains.research.code.submissions.clustering.load.distance package